app/models/manageiq/providers/vmware/infra_manager/provision_workflow/dialog_field_validation.rb

```diff
@@ -0,0 +1,15 @@
+# These methods are available for dialog field validation, do not erase.
+module ManageIQ::Providers::Vmware::InfraManager::ProvisionWorkflow::DialogFieldValidation
+  def validate_placement_host_name(field, values, dlg, fld, value)
+    result = validate_placement(field, values, dlg, fld, value)
+    return result if result.nil?
+
+    ems_cluster = EmsCluster.find_by(:id => get_value(values[:placement_cluster_name]))
+
+    if get_value(values[:placement_cluster_name]).blank? || ems_cluster.nil?
```

Comment:
Wouldn't it be enough to say `if ems_cluster.nil?` rather than calling `get_value` a second time?